### ISSUE TITLE
Add upload photo from share sheet

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/android#123]
User report on Twitter that they weren't aware of the photo uploading facility. See discussion that followed, which identified that it would be useful to add CycleStreets to the share sheet:

https://twitter.com/Kimberleytew/status/716006854233559041

**Comment by srbhmshr344** [comment]
url is not present.

**Comment by jezhiggins** [comment]
That Twitter account has been suspended.

**Comment by mvl22** [comment]
Basically, when you take a photo, there is presumably on Android (like iOS) a Share with... type sheet/menu option that lets you then post to to Twitter or Facebook.

By clicking that, it opens directly in the relevant app in the photo section with the image shown, and just whatever extra info awaiting to be added by the user.

This is a better workflow than Take photo, close camera app, find CycleStreets app, go to Photomap, click add photo, select photo from camera roll and then finally be there to add the metadata.

Basically we need to have the app registered as a photo uploading type app and then provide the route into our UI from that.
